src/dxtbx/nexus/__init__.py

```diff
@@ -104,23 +105,18 @@ def get_dxtbx_scan(
     num_images = len(scan_axis)
     image_range = (1, num_images)
 
+    oscillation = (0, 0)
     if is_rotation:
         if scan_axis.end:
             oscillation = (
                 float(scan_axis[0].to("degree").magnitude),
                 float((scan_axis.end[0] - scan_axis[0]).to("degree").magnitude),
             )
-        # elif scan_axis.increment_set:
         elif num_images > 1:
             oscillation = (
                 float(scan_axis[0].to("degree").magnitude),
                 float((scan_axis[1] - scan_axis[0]).to("degree").magnitude),
             )
-    else:
-        oscillation = (
-            0,
-            0,
-        )
```

Comment:
I welcome this change.  I happened to scan through this bit of code the other day and wished it had exactly the form you have just changed it to.

Would it be even more readable as follows?